ml/rl/workflow/identify_types_flow.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python3
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+
+from typing import Dict
+from os.path import abspath
+import pyspark
+from pyspark.sql.functions import col, collect_list, flatten, explode
+
+from ml.rl.preprocessing import normalization
+from ml.rl.workflow.types import PreprocessingOptions, TableSpec
+
+
+def get_spark_session():
```

Comment:
```suggestion

LOCAL_MASTER = "master[1]"

def get_spark_session(master=LOCAL_MASTER):
```

Later, we can add an option to override this in the workflow